Procurement/ViewModel/Filters/ForumExport/FatedUniqueBaseTypesFilter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using POEApi.Model;
+using System.Linq;
+
+namespace Procurement.ViewModel.Filters.ForumExport
+{
+    public class FatedUniqueBaseTypesFilter : IFilter
+    {
+        public FilterGroup Group
+        {
+            get { return FilterGroup.Default; }
+        }
+
+        public bool CanFormCategory
+        {
+            get { return false; }
+        }
+
+        public string Keyword
+        {
+            get { return "Fated Unique Base Types"; }
+        }
+
+        public string Help
+        {
+            get { return "Unidentified unique items that share a base type with a Fated Unique."; }
+        }
+
+        public bool Applicable(POEApi.Model.Item item)
+        {
+            Gear gear = item as Gear;
+            if (gear == null || gear.Rarity != Rarity.Unique || gear.Identified)
+                return false;
+
+            // TODO: Unique items each have their own, well, unique icon URL.  It is possible to build a map from icon
```

Comment:
A warning on this, urls for the images can change as well, I've seen this happen a few times over the years.

Reply:
I was wondering if that was the case.  Good to know!